gui-tk/godley.tcl

```diff
@@ -305,12 +305,16 @@ proc setGodleyDisplay {} {
     foreach c [info commands godleyWindow*.displayStyle] {
         $c $preferences(godleyDisplayStyle)
     }
-    foreach c [info commands godleyIcon.table.multipleEquities] {
-        $c $preferences(multipleEquities)
-    }    
     redrawAllGodleyTables
 }
-	
+
+# sets each individual Godley table mutliple equity column preference
+proc toggleEquityColumns id {
+	global preferences
+	tk_messageBox -message "Close and reopen the Godley table to enable/disable this feature" -type ok -parent .$id.table
+    set $id.multipleEquities [info commands id.godleyIcon.table.multipleEquities]
```

Comment:
I don't understand what you're trying to do here. This sets a TCL variable with the name of the multipleEquities accessor function.

Surely what you really mean here is

$id.table.multipleEquities $preferences(multipleEquities)

Reply:
Oops, that was a mistake, thanks for catching it.